src/main/java/net/minecraftforge/event/entity/living/LivingAttackEvent.java

```diff
@@ -28,10 +28,10 @@
 /**
  * LivingAttackEvent is fired when a living Entity is attacked. <br>
  * This event is fired whenever an Entity is attacked in
- * {@link EntityLivingBase#attackEntityFrom(DamageSource, float)} and
- * {@link EntityPlayer#attackEntityFrom(DamageSource, float)}. <br>
+ * {@link LivingEntity#attackEntityFrom(DamageSource, float)} and
+ * {@link net.minecraft.entity.player.PlayerEntity#attackEntityFrom(DamageSource, float)}. <br>
```

Comment:
Also fired at
[RemoteClientPlayerEntity#attackEntityFrom(DamageSource, float)](https://github.com/MinecraftForge/MinecraftForge/blob/d5cb0935e9758e1ae62dadf4535a12a5fda8e9f1/patches/minecraft/net/minecraft/client/entity/player/RemoteClientPlayerEntity.java.patch#L7) and
[ClientPlayerEntity#attackEntityFrom(DamageSource, float)](https://github.com/MinecraftForge/MinecraftForge/blob/d5cb0935e9758e1ae62dadf4535a12a5fda8e9f1/patches/minecraft/net/minecraft/client/entity/player/ClientPlayerEntity.java.patch#L7)

Reply:
Well it is the same function in the end.
Just overriden